src/main/java/Duke.java

```diff
@@ -1,10 +1,118 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        printGreetings();
+        Scanner in = new Scanner(System.in);
+        String userInput = in.nextLine();
+        executeCommand(tasks, userInput);
+        printGoodbye();
+    }
+
+    private static void executeCommand(ArrayList<Task> tasks, String userInput) {
```

Comment:
Good SLAP in this method!

Reply:
Thank you for all your kind feedback, I will make the necessary changes to my code :)